admin/dashboard.php

```diff
@@ -0,0 +1,72 @@
+<?php
+    include ("system/connection.php");
+    session_start();
+
+    if(!$_SESSION["nama_admin"]){
+        header("location:login.php");
+    }
+
+    if(!isset($_GET["page"])){
+        header("location:dashboard.php?page=berita");
+    }
+    else{
+        $page = $_GET["page"];
+    }
+?>
+<!DOCTYPE html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <link rel="stylesheet" href="css/dashboard.css">
+    <title>Dashboard - <?php echo $_SESSION["nama_admin"]; ?></title>
```

Comment:
Codacy found an issue: [Direct use of $_SESSION Superglobal detected.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)